rules/order.js

```diff
@@ -30,6 +55,7 @@ module.exports = {
         'box-sizing',
         'display',
         'overflow',
+        'vertical-align',
```

Comment:
I have no strong stance either but lets be real, `vertical-align` is the most useless property. How often do we even use it?